sycl/test/atomic_ref/add.cpp

```diff
@@ -1,6 +1,7 @@
 // RUN: %clangxx -fsycl -fsycl-unnamed-lambda -fsycl-device-only -S %s -o - \
 // RUN: | FileCheck %s --check-prefix=CHECK-LLVM
-// RUN: %clangxx -fsycl -fsycl-unnamed-lambda -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-unnamed-lambda -fsycl-targets=%sycl_triple %s -o %t.out \
+// RUN: %gpu_atomics_config
```

Comment:
The fact we have `RUN` after the line continuation looks odd to me.  I can see it's the same as the `|` on the lines above -- so it may be fine -- but I wanted to point this out so you can double-check it.

Reply:
Good note, I'll align this with https://github.com/intel/llvm-test-suite/pull/86.

Reply:
Done both here and in sub.cpp